src/Evolution/Systems/Cce/Actions/InitializeWorldtubeBoundary.hpp

```diff
@@ -57,6 +58,19 @@ struct InitializeWorldtubeBoundaryBase {
                     const ArrayIndex& /*array_index*/,
                     const ActionList /*meta*/,
                     const ParallelComponent* const /*meta*/) noexcept {
+    if constexpr (std::is_same_v<Tags::AnalyticBoundaryDataManager,
+                                 ManagerTag>) {
+      if (dynamic_cast<const Solutions::RobinsonTrautman*>(
+              &(db::get<Tags::AnalyticBoundaryDataManager>(box)
+                    .get_generator())) != nullptr) {
+        if(db::get<::Tags::TimeStepper<>>(box).number_of_substeps() != 1) {
+          ERROR(
+              "Do not use RobinsonTrautman analytic solution with a "
+              "substep-based timestepper. This is to prevent severe slowdowns "
+              "in the current RobinsonTrautman implementation");
```

Comment:
Is what these slowdowns are documented somewhere? If not, would it be useful to list them in the error message here so that if someone _really_ wants to use a substep integrator they can?